packages/reports/src/Components/Routes/__tests__/helpers.spec.tsx

```diff
@@ -1,31 +1,34 @@
-import React from 'react';
 import * as Helpers from '../helpers';
 import { routes } from '@deriv/shared';
-import getRoutesConfig from 'Constants/routes-config';
+import getRouteConfig from 'Constants/routes-config';
```

Comment:
Did you mistakenly rename this function?

Reply:
Resolved 👍🏼